_posts/2017-10-12-windows-retrospect.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: posts
+title: Bazel on Windows: a year in retrospect
+---
+
+Bazel on Windows is no longer experimental. We think Bazel on Windows is now
+stable and usable enough to qualify for the Beta status we have given Bazel on
+other platforms.
+
+Over the last year, we put a lot of work into improving Bazel's Windows support:
+
+*   **Bazel no longer depends on the MSYS runtime.** This means Bazel is now a
+    native Windows binary and we no longer link it to the MSYS DLLs. Bazel still
+    needs Bash (MSYS or Git Bash) and the GNU binutils (binaries under
+    `/usr/bin`) if your dependency graph includes `genrule` or `sh_*` rules
+    (similarly to requiring `python.exe` to build `py_*` rules), but you can use
+    any MSYS version and flavour you like, including Git Bash.
+*   **Bazel is easier to set up.** You now (typically) no longer need to set the
+    following environment variables:
+    *   `BAZEL_SH` and `BAZEL_PYTHON` -- Bazel attempts to autodetect your MSYS
+        and Python installation paths.
+    *   `JAVA_HOME` -- we release Bazel with an embedded JDK. (We also release
+        binaries without an embedded JDK if you want to use a different one.)
+*   **Visual C++ is the default C++ compiler.** We no longer use GCC by default,
+    though Bazel still supports it.
+*   **Bazel no longer uses a wrapper script for the Visual C++ compiler.**
```

Comment:
This point is hard to understand for non expert, maybe drop?

Reply:
Rephrased it.

Reply:
I'm OK with removing it. Maybe most of people don't what the wrapper script is for.

Reply:
The rephrased looks better! WDYT, @damienmg ?